purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -112,6 +112,24 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
     }
 
     // region Public Methods
+
+    /**
+     * This method will send all the purchases to the RevenueCat backend. Call this when using your own implementation
+     * for subscriptions anytime a sync is needed, like after a successful purchase.
```

Comment:
Probably should add a warning calling out that you *only* need to use this if you aren't calling `makePurchase`